src/components/checkout/app-download.tsx

```diff
@@ -0,0 +1,101 @@
+import { css } from "@/theme/css";
+import { getTranslations } from "next-intl/server";
+import { headers } from "next/headers";
+import Image from "next/image";
+import Link from "next/link";
+import { userAgent } from "next/server";
+import encodeQR from "qr";
+
+import badgeAppStore from "@/assets/badge-app-store.svg";
+import badgeGooglePlay from "@/assets/badge-google-play.svg";
+import { CenterContainer } from "../layout/center-container";
+
+export const APP_URL_APP_STORE =
+  "https://apps.apple.com/ch/app/republik/id1392772910";
+export const APP_URL_GOOGLE_PLAY =
+  "https://play.google.com/store/apps/details?id=app.republik";
```

Comment:
is this fixed enough to be hard coded? (mostly wondering about the apple id) or would it be better as env variable. This feels like env variable material to me.